products.js

```diff
@@ -177,13 +263,20 @@ class DiffStats {
         let expectedText;
         let gotText;
         if (this.feature === 'image') {
-          //compare images by src
+          //compare images by src, strip query params
 
-          // console.log(expectedDom.body.firstChild.outerHTML);
+          //console.log(expectedDom.body.firstChild.outerHTML);
           // expectedText = expectedDom.body.firstChild.outerHTML;
           // gotText = this.tuningRoutine(nodeToCssMap)(sourceDom).map(fnode => fnode.element.outerHTML)[0];
           expectedText = expectedDom.body.firstChild.src;
           gotText = this.tuningRoutine(nodeToCssMap, ...coeffs)(sourceDom).map(fnode => fnode.element.src)[0];
+          if(expectedText.includes('?')){
+            expectedText = expectedText.substr(0,expectedText.indexOf('?'));
```

Comment:
This and the above includes() call would be a good thing to factor out into a withoutQueryString() routine. Then you could avoid linearly scanning through the whole string twice for the question mark while still staying consise at the callsite.